blocks/columns/columns.js

```diff
@@ -1,18 +1,32 @@
 export default function decorate(block) {
   const cols = [...block.firstElementChild.children];
   block.classList.add(`columns-${cols.length}-cols`);
+  let downloadableColumns = false;
```

Comment:
nit: I think I wouldn't call it `downloadableColumns` but rather something more generic, since this feature could be used for all sorts of links and images